doc/mcuboot/index-ncs.rst

```diff
@@ -0,0 +1,68 @@
+.. _mcuboot_index_ncs:
+
+MCUboot
+#######
+
+.. contents::
+   :local:
+   :depth: 2
+
+MCUboot is a secure bootloader for microcontrollers.
+
+Overview
+********
+
+The goal of MCUboot is to define a common infrastructure for the bootloader, the system flash layout on microcontroller systems, and to provide a secure bootloader that enables easy software upgrade.
+
+MCUboot is operating-system and hardware independent, and relies on hardware porting layers from the operating system it works with.
+Currently MCUboot works with the various operating systems and SDKs like `Zephyr`_ and nRF Connect SDK.
+
+MCUboot is an open governance project.
+See the `membership list <https://github.com/mcu-tools/mcuboot/wiki/Members>`_ for current members, and the `project charter <https://github.com/mcu-tools/mcuboot/wiki/MCUboot-Project-Charter>`_ for more details.
+
+Content
+*******
+
+.. toctree::
+   :maxdepth: 1
+
+   release-notes.md
+   design.md
+   encrypted_images.md
+   imgtool.md
+   ecdsa.md
+   readme-zephyr.md
+   readme-ncs.rst
+   testplan-zephyr.md
+   release.md
+   SECURITY.md
+   SubmittingPatches.md
+
+Roadmap
+*******
+
+The issues being planned and worked on are tracked using GitHub issues. To give your input, visit `MCUboot GitHub Issues <https://github.com/mcu-tools/mcuboot/issues>`_.
+
+Browsing
+********
+
+You can find additional documentation on the bootloader in the source files.
+For more information, use the following links:
+* `boot/bootutil <https://github.com/mcu-tools/mcuboot/tree/main/boot/bootutil>`_: The core of the bootloader itself.
```

Comment:
Blank line above this line?

Reply:
Also, I think hyphens would work better than `:` here.

Reply:
right, fixed